src/screens/ContactScreen/index.js

```diff
@@ -195,9 +206,14 @@ const ContactScreen = ({navigation}) => {
     <SafeAreaView style={styles.container}>
       <StatusBar translucent={false} />
       <Header
-        title={StringConstant.chatTabHeaderCreateChatButtonText}
+        title={
+          sourceScreen === ANONYMOUS
+            ? StringConstant.chatTabHeaderCreateAnonChatButtonText
+            : StringConstant.chatTabHeaderCreateChatButtonText
+        }
         containerStyle={styles.containerStyle}
         subTitle={'Next'}
+        subtitleStyle={selectedUsers.length > 0 && styles.subtitleStyle(sourceScreen)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `title` prop of the `Header` component is updated to display different text based on whether the source screen is `ANONYMOUS` or not. The `subtitleStyle` prop is also updated to change color based on the source screen. These changes improve the UI by providing different headers for anonymous and non-anonymous chats.
```diff
+ title={
+   sourceScreen === ANONYMOUS
+     ? StringConstant.chatTabHeaderCreateAnonChatButtonText
+     : StringConstant.chatTabHeaderCreateChatButtonText
+ }
+ containerStyle={styles.containerStyle}
+ subTitle={'Next'}
+ subtitleStyle={selectedUsers.length > 0 && styles.subtitleStyle(sourceScreen)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->